core/domain/question_services.py

```diff
@@ -85,31 +86,117 @@ def _create_new_question(committer_id, question, commit_message):
     create_question_summary(question.id, committer_id)
 
 
-def create_new_question_skill_link(question_id, skill_id, skill_difficulty):
-    """Creates a new QuestionSkillLink model.
+def create_multi_question_skill_links_for_question(
+        user_id, question_id, skill_ids, skill_difficulties):
+    """Creates multiple new QuestionSkillLink models. Also adds
+    the skill ids to the linked skill ids of the Question model.
 
     Args:
+        user_id: str. ID of the creator.
+        question_id: str. ID of the question linked to the skill.
+        skill_ids: list(str). ID of the skills to which the question is linked.
+        skill_difficulties: list(float). The difficulty of the question with
+            respect to a skill, represented by a flot between
+            0 and 1 (inclusive).
+
+    Raises:
+        Exception. Skill difficulties and skill ids should match. The lengths of
+        the two lists are different.
```

Comment:
Add the exception about question does not exist.

Reply:
I've removed that exception from here because it is anyway done by the question model itself

Reply:
This function raises that exception as well, so it's better to also mention it in the docstring.

Reply:
Removed that exception